docs/release-notes/8.11.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[release-notes-header-8.11.0]]
+== 8.11
+
+[discrete]
+[[release-notes-8.11.0]]
+=== 8.11.0
+
+[discrete]
+[[breaking-changes-8.11.0]]
+==== Breaking changes
+* Ends support for the `filterQuery` field of the `getLiveQueryResults` and `findLiveQuery` APIs, and replaces it with the KQL field `kuery`. Requests to those APIs that used the `filterQuery` field should replace it with `kuery` ({pull}161806[#161806]).
+* In 8.11, rule APIs will only support `investigation_fields` as { field_names: string[] }. Note that, if you've added this field to your rules in 8.10, you don't need to do anything when you import your rules. 
```

Comment:
I think this example should be monospace too -- kinda awkward to read though. 🤷 Also, does the last part apply _only_ to 8.10, or is it something like "8.10 or earlier"?
```suggestion
* In 8.11, rule APIs will only support `investigation_fields` as `{ field_names: string[] }`. If you've added this field to your rules in 8.10, you don't need to do anything when you import your rules. 
```